core/modules/tests.rs

```diff
@@ -659,54 +657,6 @@ fn test_validate_import_attributes_default() {
   }
 }
 
-#[tokio::test]
```

Comment:
Moved to integration test

Reply:
I'm not sure I'm a fan of that - now we got module loading tests split between `core/modules` and `testing/checking/integration`. Can you keep it here until I land `itest!` macro and comparing `.out` files in `deno_core`?

Reply:
Meh, I guess that's fine actually.

Reply:
For sure -- I'll eventually be moving almost all of these into integration tests and we can tweak how they run at that point. There's nothing in the module tests that should prevent all of them being integration tests eventually.